docs/self-managed/identity/deployment/configuration-variables.md

```diff
@@ -47,6 +47,8 @@ For the `WEBMODELER` value, only the `KEYCLOAK_INIT_<COMPONENT>_ROOT_URL` variab
 
 ### Database configuration
 
+Identity does require a database to store information about resource authorization and multi-tenancy.
```

Comment:
Can we link to resource authorization, multi-tenancy, or resource management here?